saturn_client/core.py

```diff
@@ -0,0 +1,171 @@
+"""Python library for interacting with the Saturn Cloud API.
+
+NOTE: This is an experimental library and will likely change in
+the future
+"""
+
+import json
+import logging
+
+from sys import stdout
+from typing import Any, Dict, Optional
+from urllib.parse import urljoin
+
+import requests
+from .settings import Settings
+from ._version import get_versions
+
+__version__ = get_versions()["version"]
+
+
+logfmt = "[%(asctime)s] %(levelname)s - %(name)s | %(message)s"
+datefmt = "%Y-%m-%d %H:%M:%S"
+
+log = logging.getLogger("saturn-client")
+log.setLevel(logging.INFO)
+handler = logging.StreamHandler(stream=stdout)
+handler.setLevel(logging.INFO)
+handler.setFormatter(logging.Formatter(logfmt, datefmt))
+log.addHandler(handler)
```

Comment:
```suggestion
logfmt = "[%(asctime)s] %(levelname)s - %(name)s | %(message)s"
datefmt = "%Y-%m-%d %H:%M:%S"

log = logging.getLogger("saturn-client")
log.setLevel(logging.INFO)
handler = logging.StreamHandler(stream=stdout)
handler.setLevel(logging.INFO)
handler.setFormatter(logging.Formatter(logfmt, datefmt))
log.addHandler(handler)
```

This is _super_ nitpicky on my part, but I think it might be outstepping saturn-client's bounds to configure a logger like this. One of the banes of my devops existence is coercing all of our python libraries into logging everything in a uniform way so that we can easily parse the logs later - I'm not sure that we should inflict that on any of our customers.

Reply:
lolol this is copied directly from dask-saturn. I have no opinion.

Reply:
```suggestion
log = logging.getLogger("saturn-client")
```

Then this would be my suggestion - and I'd suggest that we make the change for dask-saturn as well. I'll make a PR for that over there - worst case, we just close it out.